PROCESS.md

```diff
@@ -24,22 +25,42 @@ Gitter or elsewhere about what happened and what was done.
 
 ### Versioning
 
-If a release is simply a bug fix, increment the patch version number
-(e.g. 1.2.3 becomes 1.2.4). These releases may happen quite quickly in
-response to reported bugs or problems, and should usually be source
-and binary compatible.
+Since `1.0.0` release, Cats adopted the *MAJOR.MINOR.PATCH* 
+[Semantic Versioning 2.0.0](http://semver.org/). In semantic versioning,
+Backward binary compatibility is maintained between *PATCH* AND *MINOR* versions.
 
-If the major version is 0, then the minor version should be updated
-(e.g. 0.2.3 becomes 0.3.0). There are no compatibility guarantees for
-this type of change.
+The main rules are:  
+* *PATCH* version Z (x.y.Z | x > 0) MUST be incremented if only backwards compatible bug fixes are introduced. 
+* *MINOR* version Y (x.Y.z | x > 0) MUST be incremented if new, binary backwards compatible functionality is introduced to the public API. It MUST be incremented if any public API functionality is marked as deprecated. 
+* Source breaking but binary compatible changes are allowed between *MINOR* versions.  
+* Binary backward breaking changes are **ONLY** allowed between *MAJOR* versions. 
+* For other scenarios, refer to [Semantic Versioning 2.0.0](http://semver.org/).
 
-If the major version is 1 or greater, then significant additions
-should increment the minor version number (e.g. 1.2.3 becomes 1.3.0)
-and breaking or incompatible changes should increment the major number
-(e.g. 1.2.3 becomes 2.0.0). These major version bumps should only
-occur after substantial review, warning, and with proper deprecation
-cycles.
 
+For a new *MINOR* version release, it's preferred to release a Release Candidate for public testing. If there are no regressions or new bugs discovered, the new *MINOR* version should be released within a couple of weeks, with no significant changes in between.    
+
+ 
+### Pre-release
+
+Before the actual release, we need to make sure all merged PRs are properly assigned with the correct Github Milestone and labels. 
+
+1. Make sure a milestone corresponding to the to-be released version exists on Github. 
+
+2. Then use the following search to gather all merged PRs to be released: 
+https://github.com/typelevel/cats/pulls?utf8=%E2%9C%93&q=merged%3A%3E2019-05-29+
+replace `2019-05-29` with the last release date.
+
+3. For PRs that add no value to the code or documentation or build, for example, community announcements or additions to adopter list, we do not include them in the release notes. Assign these PRs a special milestone: `Excluded from release notes` 
+
+4. Assign the rest PRs with the target milestone and one or more of the following labels: `testing`, `bug`, `build`, `documentation`, `enhancement`, `Source Breaking` and `Binary Breaking`.
```

Comment:
Is it too late to be consistent about case for labels? I know it's silly but looking at this list is extremely painful to me.

Reply:
IIRC, it's easy to change label text retrospectively. However, the intention for the case choice for breaking-change labels is for them to stand out as they often deserve more attention from users. We used color as well but there are users that are color blinded.